Formula/systemd.rb

```diff
@@ -0,0 +1,56 @@
+class Systemd < Formula
+  desc "System and service manager"
+  homepage "https://wiki.freedesktop.org/www/Software/systemd/"
+  url "https://github.com/systemd/systemd/archive/v246.tar.gz"
+  sha256 "4268bd88037806c61c5cd1c78d869f7f20bf7e7368c63916d47b5d1c3411bd6f"
+  license all_of: ["GPL-2.0-or-later", "LGPL-2.1-or-later"]
+  head "https://github.com/systemd/systemd.git"
+
+  depends_on "coreutils" => :build
+  depends_on "docbook-xsl" => :build
+  depends_on "gettext" => :build
+  depends_on "gperf" => :build
+  depends_on "intltool" => :build
+  depends_on "libgpg-error" => :build
+  depends_on "libtool" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libcap"
+  depends_on :linux
+  depends_on "lz4"
+  depends_on "openssl@1.1"
+  depends_on "util-linux" # for libmount
+  depends_on "xz"
+
+  uses_from_macos "libxslt" => :build
+  uses_from_macos "m4" => :build
+  uses_from_macos "expat"
```

Comment:
A line `depends_on :linux` and multiple usages of `uses_from_macos` looks a bit strange (or rather inconsistent) to me. Shall we change `uses_from_macos` to `depends_on`?

Reply:
Yes makes sense. I'll rebase and make the change when util-linux builds.

Reply:
It's not allowed:
```
2021-04-03T21:30:48.8425000Z systemd:
2021-04-03T21:30:48.8427640Z   * Dependency 'libxslt' is provided by macOS; please replace 'depends_on' with 'uses_from_macos'.
2021-04-03T21:30:48.8431270Z   * Dependency 'm4' is provided by macOS; please replace 'depends_on' with 'uses_from_macos'.
2021-04-03T21:30:48.8434630Z   * Dependency 'expat' is provided by macOS; please replace 'depends_on' with 'uses_from_macos'.
```